projects/template.md

```diff
@@ -57,20 +82,65 @@
 
 ## Milestones
 
-> **Instructions:** What are the milestones that will be delivered, and at each milestone: 1. What new things will be delivered which the Network Steward can use to verify that the milestone was a success? 2. How much of the budget should be paid out?
+> **Instructions:** What are the milestones that will be delivered, and at each milestone:
+> 1. What new things will be delivered which the Network Steward can use to verify that the milestone was a success?
+> 2. How much of the budget should be paid out?
+>
+> If you have already done work on the project, create *negative* milestones and write success criteria for
+> these milestones in order to justify what has already been done.
 
 ### Milestone 0 (Kickoff)
 
-> **Instructions:** Explain how much PKT must be provided at the beginning of the project and any pre-existing deliverables which will be immediately delivered. **NOTE:** Projects which ask for more of their budget up-front will be regarded as more risky by the Network Steward, for details see the [acceptance_process](https://github.com/pkt-cash/ns-projects/blob/master/acceptance_process.md).
+> **Instructions:** Document anything which you did on the project before the project began, any prototypes,
+> mockups, planning or useful work which was done. This should corrispond to the **Pre-project effort**
+> defined at the beginning of the document.
+>
+> Then explain how much PKT must be provided at the beginning of the project. **NOTE:** Projects which ask for larger advances with less pre-project work will be regarded as more risky, for details see the [acceptance_process](https://github.com/pkt-cash/ns-projects/blob/master/acceptance_process.md).
 
 After the project is accepted, the Network Steward will pay XXX PKT
 
 ### Milestone 1
 
-> At this point, a user will be able to ...
+#### Success criteria
+> **Instructions**: This is where you write success criteria which will be used for verifying that
+milestone 1 was delivered correctly.
 
-> After a report is delivered on this milestone and it is approved by the NS, the NS will send XXX PKT.
+#### Payout
+After a report is delivered on this milestone and it is approved by the NS, the NS will send **TODO** PKT.
 
 ### Milestone 2
 
-...
\ No newline at end of file
+...
+
+## Disclosure
+I hereby submit this application in good faith and I attest that I have made no effort, nor do I
+intend to make effort, influence the Network Steward to accept this or any other project I have
+submitted.
+
+*Please check one or more:*
+
+1. Conflicts
+  1. [ ] A company is receiving the funds
```

Comment:
The word “company” is vague here - suggest we maybe want to use “organisation” instead and specify that it may also cover private enterprise, charities, etc.

Reply:
Changed, I'm fine with "Organization"